vmtest-haskell

```diff
@@ -1,12 +1,20 @@
 #!/bin/sh
 
-KRUN_BINARY=".build/k/k-distribution/target/release/k/bin/krun"
-KOMPILED_DEF=".build/haskell/"
-HASKELL_BACKEND_CMD=".build/kore/bin/kore-exec"
+PGM=$1
 
-cSCHEDULE=DEFAULT
-cMODE=VMTESTS
+assembled="$(mktemp)"
+kast="$(mktemp)"
+output="$(mktemp)"
+trap "rm -rf $kast $output $assembled" INT TERM EXIT
+"$(dirname "$0")/assemble-iele-test" "$PGM" > "$assembled" || exit $?
+"$(dirname "$0")/kore-json.py" "$assembled" '`DEFAULT`(.KList)' '`VMTESTS`(.KList)' > "$kast"
 
-PGM=$1
+kore-exec .build/haskell/iele-testing-kompiled/definition.kore --pattern "$kast" --module IELE-TESTING --smt none --output "$output"
```

Comment:
Will `kore-exec` be on `PATH`?

Reply:
```suggestion
! kore-exec .build/haskell/iele-testing-kompiled/definition.kore --pattern "$kast" --module IELE-TESTING --smt none --output "$output" || exit 0
```

Reply:
Yes, when you run the make target kore-exec will be on the PATH.

Reply:
I'm leaving the original behavior of exiting when the code is 0 and running the kprint command otherwise.